__tests__/app.test.js

```diff
@@ -179,4 +179,110 @@ describe("GET api/reviews/2", () => {
         );
       });
   });
+});
+
+describe("GET /api/reviews", () => {
+  test("200: should return an array of reviews", () => {
+    return request(app)
+      .get("/api/reviews")
+      .expect(200)
+      .then(({ body }) => {
+        // expect(body.reviews).toHaveLength(13);
+        body.reviews.forEach((review) => {
+          expect(review).toEqual(
+            expect.objectContaining({
+              title: expect.any(String),
+              designer: expect.any(String),
+              owner: expect.any(String),
+              review_img_url: expect.any(String),
+              review_body: expect.any(String),
+              review_id: expect.any(Number),
+              category: expect.any(String),
+              created_at: expect.any(String),
+              votes: expect.any(Number),
+            })
+          );
+        });
+      });
+  });
+});
+
+
+describe("GET /api/reviews?category=euro game", () => {
+  test("200: should return an array of reviews", () => {
+    return request(app)
+      .get("/api/reviews?category=euro game")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.reviews).toHaveLength(1);
+        body.reviews.forEach((review) => {
+          expect(review).toEqual(
+            expect.objectContaining({
+              title: expect.any(String),
+              designer: expect.any(String),
+              owner: expect.any(String),
+              review_img_url: expect.any(String),
+              review_body: expect.any(String),
+              review_id: expect.any(Number),
+              category: expect.any(String),
```

Comment:
here you can have `category: "euro game"` because we're only expecting reviews with the "euro game" category

Reply:
Ah yes, thank you :)